box2dplugin.h

```diff
@@ -33,7 +33,8 @@
  */
 class Q_DECL_EXPORT Box2DPlugin : public QQmlExtensionPlugin
 {
-    Q_OBJECT
```

Comment:
removing this makes an error when trying to run the examples (after I copy the libBox2D.qml and qmldir in a qml/Box2D folder):
```bash
> qml6 -I ${PWD}/qml examples/ui.qml 
Failed to extract plugin meta data from '.../qml-box2d/qml/Box2D/libBox2D.so': '.../qml-box2d/qml/Box2D/libBox2D.so' is not a Qt plugin (metadata not found)
```

Reply:
I removed it due to compile issues with MSVC. MinGW and GCC builds work fine with Q_OBJECT. I suppose it should not be removed.

Reply:
Which compile issues were those? Talking about compile issues, I'll have a look at setting up GitHub Actions.

Reply:
MSVC does generate staticMetaObject in moc_box2dplugin.cpp resulting in 'main.cpp.obj:-1: error: LNK2001: unresolved external symbol "public: static struct QMetaObject const Box2DPlugin::staticMetaObject" (?staticMetaObject@Box2DPlugin@@2UQMetaObject@@B)'.
I haven't dedicated much time to resolve this problem, since I am not very familiar with Qbs.